python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -129,28 +151,60 @@ def makePlot(
         # set up figure
         fig = plt.figure(dpi=300)
         hist_fig, side_fig = fig.subfigures(1, 2, wspace=0, width_ratios=[3, 1])
-        axs = self._makeAxes(hist_fig)
+        axs, ncols, nrows = self._makeAxes(hist_fig)
 
         # loop over each panel; plot histograms
-        cols = self._assignColors()
-        all_handles, all_nums, all_meds, all_mads = [], [], [], []
+        colors = self._assignColors()
+        nth_panel = len(self.panels)
+        nth_col = ncols
+        nth_row = nrows - 1
         for panel, ax in zip(self.panels, axs):
-            nums, meds, mads = self._makePanel(data, panel, ax, cols[panel], **kwargs)
+            nth_panel -= 1
+            nth_col = ncols - 1 if nth_col == 0 else nth_col - 1
+            if nth_panel == 0 and nrows * ncols - len(self.panels) > 0:
+                nth_col -= 1
+            # Set some font sizes based on number of panels being plotted.
+            label_font_size = max(6, 10 - nrows)
+            legend_font_size = max(4, int(8 - len(self.panels[panel].hists) / 2 - nrows // 2))  # type: ignore
```

Comment:
does this need a `# type: ignore` here? Looks fine on my side?

Reply:
I get:

```
$ mypy python/lsst/analysis/tools/actions/plot/histPlot.py 
python/lsst/analysis/tools/actions/plot/histPlot.py:196: error: <nothing> has no attribute "hists"  [attr-defined]
Found 1 error in 1 file (checked 1 source file)
```
if I take it out, so leaving it as is.